scan/src/components/ScriptExecute.re

```diff
@@ -83,17 +117,17 @@ let reducer = _state =>
 [@react.component]
 let make = (~script: ScriptHook.Script.t) => {
   let params = script.info.params;
-  let preData = params->Belt.List.map(({name}) => (name, ""));
+  let preData = params->Belt.List.map(({name, dataType}) => (name, dataType, ""));
```

Comment:
Why is data type necessary here? Don't see how it's being used.

Reply:
NVM. it's being used when creating `parameterInput`. Though I think it makes more sense to just use params directly, maybe using `Belt.List.zip`? Up to you.